src/Android/Xamarin.Android/Samples/Analysis/LineOfSightLocation/LineOfSightLocation.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.UI.Controls;
+using Esri.ArcGISRuntime.UI.GeoAnalysis;
+using System;
+
+namespace ArcGISRuntimeXamarin.Samples.LineOfSightLocation
+{
+    [Activity]
+    public class LineOfSightLocation : Activity
+    {
+        // Create and hold reference to the used MapView
+        private SceneView _mySceneView = new SceneView();
+
+        // URL for an image service to use as an elevation source
+        private string _elevationSourceUrl = @"http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
+
+        // Location line of sight analysis
+        private LocationLineOfSight _lineOfSightAnalysis;
+
+        // Observer location for line of sight
+        private MapPoint _observerLocation = null;
```

Comment:
I don't know if these need to be explicitly set to null, I think we usually don't.